### ISSUE TITLE
Avis home page collapsible menu once opened can not be closed.

### ISSUE THREAD

**Issue by aishahussain24** [Kishan1601/Test-1#5]
Environment
Testing Device
Laptop/PC
Operating System
Windows 10
Browser Name And Version
Chrome Version 98.0.4758.102OfficialBuild
Screen Reader Name  And Version
NVDA Version2021.3.5

Repro Steps
1. Tab to navigate to the collapsible menu option list.
2. Open one of the menu by pressing enter.
3. Try closing the menu with the keyboard.

Expected Result
.The menu should be operable i.e. it should get closed with the keyboard

Actual Result
The menu is not getting closed with the keyboard

WCAG Guideline
https://www.w3.org/TR/WCAG21/#keyboard
